### PR TITLE
School Info Completeness:  submit sii form with only non-US country

### PR DESCRIPTION
**Pull request by nkiruka**
[Bug bash](https://docs.google.com/document/d/1KxMHcnQlTCKWJxHi0mK2uh_dBSIcqvWCe_4vTd7OCYI/edit#bookmark=id.r3p047rsnb08)

Bug re-pro
Select non-US country
Click “Save”
Required messages show up for Type and City/Town
Once a type is selected, City/Town is no longer required and user can Save out of the interstitial

Fix:  School type and city are not required when a non-US country is selected
Remove error messages for school type and city/town

Before
us_to_non_us_before

After
us_to_non_us_after

### CODE REVIEW

File: apps/src/lib/ui/SchoolInfoInterstitial.jsx [hunk 1]
```diff
@@ -218,7 +218,7 @@ export default class SchoolInfoInterstitial extends React.Component {
       errors.country = true;
       isValid = false;
     } else {
-      if (this.isBlank(schoolType)) {
+      if (this.isBlank(schoolType) && country === 'United States') {
```
Comment (bethanyaconnor) [inline apps/src/lib/ui/SchoolInfoInterstitial.jsx]: There is a check below for the country as well. Could we make line 220 `else if (country === 'UnitedStates')`?

**Review by islemaster — APPROVED** [review]
:+1: Great minimal fix and quick test case!  Looks good - I also agree with Bethany's suggestion to simplify the validation logic.

File: apps/src/lib/ui/SchoolInfoInterstitial.jsx [hunk 2]
```diff
@@ -217,7 +217,7 @@ export default class SchoolInfoInterstitial extends React.Component {
     if (this.isBlank(country)) {
       errors.country = true;
       isValid = false;
-    } else {
+    } else if (country === 'United States') {
```
Comment (bethanyaconnor) [inline apps/src/lib/ui/SchoolInfoInterstitial.jsx:220]: nit: you can remove the `if (country === 'United States')` line on line 225.

**Review by bethanyaconnor — APPROVED** [review]

File: apps/src/lib/ui/SchoolInfoInterstitial.jsx (hunk 2 above)
Comment (nkiruka) [reply apps/src/lib/ui/SchoolInfoInterstitial.jsx:220]: Thanks!

**Pull request merged**